### PR TITLE
[#224] Add support for SVG and color assets

### PR DESCRIPTION
**Pull request by chornerman**
- Solves #224

## What happened 👀

- [x] Set up `flutter_svg` integration for adding SVG support
- [x] Set up `colors.xml` support
- [x] Add integration examples for colors and SVG

## Insight 📝

N/A

## Proof Of Work 📹

Android

224_android

iOS

224_ios

### CODE REVIEW

File: bricks/template/__brick__/{{project_name.snakeCase()}}/android/build.gradle [hunk 1]
```diff
@@ -6,7 +6,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.1.3'
```
Comment (luongvo) [inline bricks/template/__brick__/{{project_name.snakeCase()}}/android/build.gradle:9]: Please avoid adding unrelated changes to this task. I think you should base this PR on my one https://github.com/nimblehq/flutter-templates/pull/201 to extend the updates.

File: sample/lib/main.dart [hunk 2]
```diff
@@ -73,15 +74,18 @@ class HomeScreen extends StatelessWidget {
...
             ),
             const SizedBox(height: 24),
             Text(AppLocalizations.of(context)!.hello),
             Text(
               FlutterConfig.get('SECRET'),
-              style: const TextStyle(color: Colors.black, fontSize: 24),
+              style: const TextStyle(
+                color: ColorName.nimblePrimaryBlue,
+                fontSize: 24,
+              ),
```
Comment (luongvo) [inline sample/lib/main.dart:88]: We should not commit & push changes for the sample project. It's automatically managed by the [generate_sample](https://github.com/nimblehq/flutter-templates/blob/develop/.github/workflows/generate_sample.yml) workflow.

File: bricks/template/__brick__/{{project_name.snakeCase()}}/lib/main.dart [hunk 3]
```diff
@@ -4,6 +4,7 @@ import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 import 'package:{{project_name.snakeCase()}}/gen/assets.gen.dart';
 import 'package:go_router/go_router.dart';
 import 'package:package_info_plus/package_info_plus.dart';
+import 'package:{{project_name.snakeCase()}}/gen/colors.gen.dart';
```
Comment (luongvo) [inline bricks/template/__brick__/{{project_name.snakeCase()}}/lib/main.dart:7]: could you please reorder the import to below of the `import 'package:{{project_name.snakeCase()}}/gen/assets.gen.dart';`?

**Review by luongvo — COMMENTED** [review]
The rest lgtm

**Review by markgravity — APPROVED** [review]

File: bricks/template/__brick__/{{project_name.snakeCase()}}/ios/Podfile.lock [hunk 4]
```diff
@@ -31,4 +31,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 22cb4141f6291ab800e0ca928cc1f87e9a69bf64
 
-COCOAPODS: 1.11.2
+COCOAPODS: 1.11.3
```
Comment (doannimble) [inline bricks/template/__brick__/{{project_name.snakeCase()}}/ios/Podfile.lock:34]: @chornerman it is not a big due, could we update the latest version? If not, please do not make the change 🙏

**Review by doannimble — COMMENTED** [review]
rest lgtm

File: bricks/template/__brick__/{{project_name.snakeCase()}}/lib/main.dart (hunk 3 above)
Comment (chornerman) [reply bricks/template/__brick__/{{project_name.snakeCase()}}/lib/main.dart]: Fixed in https://github.com/nimblehq/flutter-templates/pull/231/commits/be37d7314d53fd11aadbe897625eadcc86c9e40e

File: bricks/template/__brick__/{{project_name.snakeCase()}}/ios/Podfile.lock (hunk 4 above)
Comment (chornerman) [reply bricks/template/__brick__/{{project_name.snakeCase()}}/ios/Podfile.lock:34]: @doannimble I do not update it manually. It's auto-generated when we run the project on iOS. So I think we should leave it as is 🙏🏼

**Review by luongvo — APPROVED** [review]

**Review by doannimble — APPROVED** [review]
